_pages/about-us/history-and-values.md

```diff
@@ -53,7 +53,7 @@ Like Lean Startup, we favor experimentation, customer feedback, analytics, and i
 
 - **Be the change.** We intend to lead by example, by instruction, and with hands-on assistance.
 - **Think like a designer.** We believe that user-centered design can fundamentally change the experience the public has with government. We build only what people really need, nothing more. User needs are the driver for all decisions — not stakeholder or government needs.
-- **Data-driven.** We use metrics and analytics to augment our user research. We measure everything, including ourselves. 18F does more than make websites; we enable the discovery of information. Whenever possible, we think “API first” and lead with data.
+- **Data-informed.** We collect qualitative and quantitative data to inform our decisions. We measure everything, including ourselves. 18F does more than make websites; we enable the discovery of information. Whenever possible, we think “API first”.
```

Comment:
What prompted this change, Andrew? I think the prior language was _arguably_ clearer and more straightforward, so I'm trying to get a better sense of where we're trying to go with this change.

On a less philosophical level, I would strongly suggest changing the last sentence to "Whenever possible, we think "data-first", because an API is one of many possible solutions, and using that particular solution may or may not be supported by the available data. In other words, in some cases the gathered qualitative and quantitative data could tell us DON'T do this via API.

Reply:
Ah semantics. I'm happy to concede this point (my goal is really to just change us to being data-informed vs. data-driven), but user research is more than qualitative data — you can collect quantitative data by surveying your users, for example. User research is a set of activities for understanding the user's experience. If you'd prefer, we can keep what we have now ("We use metrics and analytics to augment our user research") so long as we update it to say "design research," since that's what we're calling it in the UX guide.

For (2) can we open a separate issue for the API-first phrasing? I don't mind that suggestion, but would like to update one word so that our UX team principles better cohere with the 18F-wide ones.

Reply:
Updating this per our conversation, @tbaxter-18f